commit 7c261cbc8465973ef9b45af60be63a246d5e8062
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:22:23 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-ignore-errors-in-postconfigure_checks.diff

diff --git a/src/port1.0/portconfigure.tcl b/src/port1.0/portconfigure.tcl
index 93e12fbaf..94957b8a2 100644
--- a/src/port1.0/portconfigure.tcl
+++ b/src/port1.0/portconfigure.tcl
@@ -1974,7 +1974,7 @@ proc portconfigure::check_warnings {warning_flag} {
 
     set files [list]
 
-    fs-traverse -tails file [list ${workpath}] {
+    fs-traverse -tails -ignoreErrors file [list ${workpath}] {
         if {[file tail $file] in [list config.log CMakeError.log meson-log.txt] && [file isfile [file join ${workpath} $file]]} {
             # We could do the searching ourselves, but using a tool optimized for this purpose is likely much faster
             # than using Tcl.
@@ -2025,7 +2025,7 @@ proc portconfigure::check_implicit_function_declarations {} {
     # Map from function name to config.log that used it without declaration
     set undeclared_functions [dict create]
 
-    fs-traverse -tails file [list ${workpath}] {
+    fs-traverse -tails -ignoreErrors file [list ${workpath}] {
         if {[file tail $file] in [list config.log CMakeError.log meson-log.txt] && [file isfile [file join ${workpath} $file]]} {
             # We could do the searching ourselves, but using a tool optimized for this purpose is likely much faster
             # than using Tcl.
